src/main/java/hudson/security/ProjectMatrixAuthorizationStrategy.java

```diff
@@ -36,7 +36,6 @@
 import com.thoughtworks.xstream.converters.UnmarshallingContext;
 import com.thoughtworks.xstream.mapper.Mapper;
 import com.thoughtworks.xstream.core.JVM;
-import hudson.PluginManager;
```

Comment:
gratuitous

Reply:
In the first commit of four, I'd rather keep the history around than rewrite it to not clean this file up.